frontend/src/components/utils/tags/TagEdition.tsx

```diff
@@ -100,7 +99,8 @@ export const TagEdition = ({ model, id }) => {
                     `Erreur durant le chargement : ${error.toString()}`
                 );
             });
-    }, [model, id]);
+        // eslint-disable-next-line
```

Comment:
Please explain each time you are doing this
I don't understand why it happens so many times